public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -135,6 +137,7 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
     this.database = instanceSettings.database;
     this.basicAuth = instanceSettings.basicAuth;
```

Comment:
Looking into migrating this, do you think this + other headers need to be migrated?